app/mailers/notification_mailer.rb

```diff
@@ -71,6 +73,14 @@ def set_account
     @account = @notification.from_account
   end
 
+  def list_headers
+    headers['List-ID'] = "<#{@type}.#{Rails.configuration.x.local_domain}>"
+    headers['List-Archive'] = "<#{web_url('notifications')}>"
+    headers['List-Subscribe'] = "<#{settings_preferences_notifications_url}>"
+    headers['List-Unsubscribe'] = "<#{@unsubscribe_url}>"
+    headers['List-Unsubscribe-Post'] = 'List-Unsubscribe=One-Click'
```

Comment:
I'm not sure we want *all* of these headers. Also the `List-ID` could possibly use the username, as someone may have multiple account on the same server.

Reply:
Ok, I have removed `List-Archive` and `List-Subscribe`. We can easily add them again if popular email clients adds support for them.

Good point about `List-ID`. It now also includes the username.